tests/unit/Actions.spec.ts

```diff
@@ -519,7 +519,7 @@ describe('Actions component - Filing Functionality', () => {
     expect(mockBuildFiling).toHaveReturned()
 
     expect(mockUpdateFiling).toHaveBeenCalledWith('T1234567', filing, false)
-    // expect(mockUpdateFiling).toHaveReturned()
+    expect(mockUpdateFiling).toHaveLastReturnedWith(padErrorFiling)
```

Comment:
This is a better test than toHaveReturned... But does toHaveReturned not work?

Reply:
The test fails but the expected output is a weird array that includes the padErrorFiling. So, I thought using this would be nicer and easier.

Reply:
I agree. Resolving.

Reply:
One more comment: I generally like to understand why something that I think should work, doesn't. But not today.